src/core/generator/from_json.rs

```diff
@@ -1,41 +1,68 @@
+use std::collections::HashMap;
+
+use convert_case::{Case, Casing};
+use derive_getters::Getters;
 use serde_json::Value;
 use url::Url;
 
-use super::json::{self, TypesGenerator};
+use super::json::{self, GraphQLTypesGenerator};
 use super::NameGenerator;
-use crate::core::config::Config;
+use crate::core::config::{Config, GraphQLOperationType};
+use crate::core::http::Method;
 use crate::core::merge_right::MergeRight;
 use crate::core::transform::{Transform, TransformerOps};
 use crate::core::valid::{Valid, Validator};
 
+#[derive(Getters)]
```

Comment:
Can make all fields public and drop the getters

Reply:
done